test/integration/controller_test.go

```diff
@@ -164,11 +163,11 @@ func TestBasicFlows(t *testing.T) {
 			*instance.Spec.OSBDashboardURL,
 		)
 	}
-	if *retInst.Spec.OSBLastOperation != *instance.Spec.OSBLastOperation {
+	if retInst.Status.LastOperation != testLastOperation {
```

Comment:
Do we actually expect to get a last operation back here?  I had expected this to be exercising a sync provision, because I didn't remember it changing in your last PR.